packages/components/addon/components/hds/pagination-bar/index.hbs

```diff
@@ -1,4 +1,85 @@
 {{! ADD YOUR TEMPLATE CONTENT HERE }}
 <div class={{this.classNames}} ...attributes>
-  {{yield}}
-</div>
\ No newline at end of file
+  <div class="hds-pagination-bar__total-count">1–10 of 100</div>
+
+  <nav class="hds-pagination-bar__pagination" aria-label="Pagination navigation">
+    <ul class="hds-pagination-bar__page-list">
+      <li class="hds-pagination__page hds-pagination__page--previous">
+        <a class="hds-pagination__page-link" aria-label="Previous page" href="/page-?">
```

Comment:
Do you know if they use query params for this?

Reply:
The href values are just placeholders at this point.